protos/escrow.proto

```diff
@@ -0,0 +1,99 @@
+syntax = "proto3";
+import "google/protobuf/timestamp.proto";
+
+package btfs;
+
+service EscrowService{
+	rpc submitContracts(EscrowContractRequest) returns (EscrowContractResult) {}
```

Comment:
`SubmitContracts` upper case letters for all service functions

Reply:
fixed